protocols/enocean/org.eclipse.smarthome.protocols.enocean.basedriver.impl/enocean_config/enocean_config.txt

```diff
@@ -0,0 +1,8 @@
+#Wed Dec 17 17:11:53 CET 2014
+#0x00112233_RORG_FUNC_TYPE_FRIENDLYNAME_DESCRIPTION=aa#bb#cc##dd
```

Comment:
> Is it ok for ESH?

I just do not think it is ok for an official distributable, which is meant to be included in products. If it is required by the test/sample client, it should imho a part of this test scenario, e.g. offered as a fragment that can be added for this purpose, don't you think?